src/Native/CMakeLists.txt

```diff
@@ -265,3 +263,10 @@ endif(NOT CLR_CMAKE_PLATFORM_WASM)
 if(NOT WIN32)
   add_subdirectory(System.Private.CoreLib.Native)
 endif(NOT WIN32)
+
+# Build ObjWriter on Linux only
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+  if($ENV{__ObjWriterBuild})
+    add_subdirectory(ObjWriter/llvmCap)
```

Comment:
Is the Cap suffix a convention used for similar external projects? I do not see a problem with the name ... just wondering where it came from.

Reply:
No, maybe, any suggestions?)